WS2012R2/lisa/xml/STOR_VSS_Backup_Tests.xml

```diff
@@ -272,6 +273,30 @@
             <timeout>3000</timeout>
             <OnError>Continue</OnError>
         </test>
+
+        <test>
+            <testName>STOR_VSS_Backup_Disable_Enable_VSS</testName>
+            <setupScript>setupscripts\RevertSnapshot.ps1</setupScript>
+            <files>remote-scripts/ica/utils.sh</files>
+            <testScript>setupscripts\STOR_VSS_Backup_Disable_Enable_VSS.ps1</testScript>
+            <testParams>
+                <param>TC_COVERED=VSS-21</param>
+            </testParams>
+            <timeout>2400</timeout>
+            <OnError>Continue</OnError>
+        </test>
+
+        <test>
```

Comment:
Here is two test scenarios.
The first case is disable/enable backup integration service, the second case is stop hypervvssd daemons from terminal.
I'll update case name and file name to make them different, also add checking "start hypervvssd to backup after stop". 

In order to make logic clear, suggest not combine together.